### ISSUE TITLE
Verification email is not being sent

### ISSUE THREAD

**Issue by MunhozThiago** [zenodo/zenodo#2176]
Hello,

I'm trying to upload a contribution but after my signup, the verification email is never sent. Tried already to use other accounts and hit the button to resend the verification email, nothing seems to have an effect.

**Comment by nshozi69** [comment]
For me, after I registered the email showed as text on the window that I ran "zenodo run" on. The email shows up there. You do not get an actual email. I assume you first need to do settings for the email/smtp before it can send actual email.

**Comment by nshozi69** [comment]
> 
> 
> Hello,
> 
> I'm trying to upload a contribution but after my signup, the verification email is never sent. Tried already to use other accounts and hit the button to resend the verification email, nothing seems to have an effect.

For me, after I registered the email showed as text on the window that I ran "zenodo run" on. The email shows up there. You do not get an actual email. I assume you first need to do settings for the email/smtp before it can send actual email.

**Comment by myoung02** [comment]
I'm having the same issue. I've logged into Zenodo using my ORCID account and the button to send a verification email produces no effect. I have no idea what "zenodo run" means...

**Comment by tengku-hanis** [comment]
I'm having the same issues. I signed up to Zenodo using my ORCID account as well.

**Comment by myoung02** [comment]
> 
> 
> I'm having the same issues. I signed up to Zenodo using my ORCID account as well.

I've discovered that this, and several other features of Zenodo do not work on Firefox, but were fully function when I switched to Chrome. I suggest using a different browser and your issue will likely be solved!

**Comment by nshozi69** [comment]
Awesome. I will aso test this out!

**Comment by priyaaarora1** [comment]
Hello,
I am trying to "request access" to the https://zenodo.org/record/6475787 for Muse challeneg dataset as suggested by one of the organizers. I am not logged-in while sending the request, however, the screen does state to check email for instructions to follow. However, I do not see any such email in my account. I tried with chrome "Version 96.0.4664.110 (Official Build) (x86_64)", Firefox latest version, and incognito mode of Chrome too. May I know why I am not getting that instruction email?. 
<img width="902" alt="Screen Shot 2022-06-05 at 10 44 32 AM" src="https://user-images.githubusercontent.com/107092975/172527993-1a8ab73d-94a1-4f0e-b959-753bfeba42cd.png">

**Comment by w990417** [comment]
Have the same issue. Tried signing up with email and with GitHub. Tried many browsers on both windows and mac and the issue persists.

**Comment by jonathanlilly** [comment]
Check your Spam folder; that's where I found mine

**Comment by slanterns** [comment]
I tried 2 different email accounts and cannot receive any confirmation email. I've also checked the spam folder.

**Comment by PascalIversen** [comment]
I also did not get a confirmation email. Also not in the spam folder.

**Comment by slanterns** [comment]
It may takes ~30min before the email is actually sent.

**Comment by PascalIversen** [comment]
@slanterns well thats unfortunate because TIL the Link in the confirmation Email expires after 30 min.

**Comment by slanterns** [comment]
Yeah, and my only solution is try more times until you are in luck to receive it within 30min :(

**Comment by carlinmack** [comment]
BTW you can resend the confirmation email using https://zenodo.org/confirm

**Comment by redst4r** [comment]
It takes indeed 15-20min for the email to get into your inbox, you'll need some patience :)
